geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnectionFactory.java

```diff
@@ -71,12 +76,16 @@ public ServerConnection makeServerConnection(Socket socket, InternalCache cache,
     }
   }
 
+  private int readProtocolVersionByte(Socket socket) throws IOException {
```

Comment:
I think this would be just as clear (or clearer) inlined.

Reply:
either way

Reply:
I like having this as a separate function, so since there doesn't seem to be a strong consensus for changing it I'll just leave it as is.